datalake/src/main/java/com/sequenceiq/datalake/cm/ClouderaManagerProxiedClientFactory.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.datalake.cm;
+
+import com.cloudera.api.swagger.client.ApiClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class ClouderaManagerProxiedClientFactory {
+
+    private static final String API_VERSION_32 = "v32";
```

Comment:
why this version? in CB we use 31 or 40. I think this should be in sync

Reply:
Updated to 31